src/ChartAtom.tsx

```diff
@@ -1,23 +1,40 @@
 import React from 'react';
 import { css } from 'emotion';
 import { space } from '@guardian/src-foundations';
-import { neutral, text } from '@guardian/src-foundations/palette';
+import { text } from '@guardian/src-foundations/palette';
 
 import { ChartAtomType } from './types';
 
-export const ChartAtom = ({ id, url }: ChartAtomType): JSX.Element => (
-    <div
-        data-atom-id={id}
-        data-atom-type="chart"
-        className={css`
-            padding-bottom: ${space[1]}px;
-            padding-left: ${space[2]}px;
-            padding-right: ${space[2]}px;
-            border-top: 1px solid ${text.primary};
-            color: ${text.primary};
-            background: ${neutral[97]};
-        `}
-    >
-        <iframe src={url} frameBorder="0" height="450px"></iframe>
-    </div>
-);
+export const ChartAtom = ({ id, url, html }: ChartAtomType): JSX.Element => {
+    // The script tag needs to replaced with 'gu-script' to work with iframe
+    // resizing on frontend
+
+    html = html
+        .split('<script>')
+        .join('<gu-script>')
+        .split('</script>')
+        .join('</gu-script>');
+    return (
+        <div
+            data-atom-id={id}
+            data-testid="chart"
+            data-atom-type="chart"
+            data-snippet-type="chart"
+            className={css`
+                padding-bottom: ${space[1]}px;
+                padding-left: ${space[2]}px;
+                padding-right: ${space[2]}px;
+                color: ${text.primary};
+                margin-bottom: ${space[2]}px;
```

Comment:
Are the paddings and margins definitely required for DCR? Certainly the right and left paddings should be handled by the article gutters I think?